content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -431,6 +431,14 @@ Used on: PersistentVolumeClaim
 
 This annotation has been deprecated.
 
+### volume.beta.kubernetes.io/storage-class (deprecated)
+
```

Comment:
```suggestion

Example: `volume.beta.kubernetes.io/storage-class: "example-class"`

```

Reply:
@sftim I have taken this suggestion as a new commit. Thanks a lot for taking time to review the changes.